srcpkgs/osmin/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'osmin'
+pkgname=osmin
+version=1.8.7
+revision=1
+_libosmscout_version=1.9.2
+archs="~armv6l-musl ~armv7l ~i686" # libomp-devel
```

Comment:
how about duplicating the list for clarity? https://github.com/void-linux/void-packages/blob/d21de3397f4b3f9ccf738e3cfa7ab5f48adc127c/srcpkgs/llvm12/template#L64

Reply:
I'm not sure about this. In llvm12 the variable `_libomp_enable` is used afterwards to create subpackages. I think this is not needed here.

Reply:
first, your list is incomplete as is (should use starred globs for `armv[67]`.

second, your point of restricting archs is to break early and not fail mid dependency installation step due to missing dependency `libomp-devel` , isn't it?
so it's technically incorrect to mix whitelist and blacklist this way, as using the blacklist like you do here makes it try build for archs that don't support libomp per llvm12's whitelist, for example `ppc`, `mips*`, `armv5*`.

so you either blacklist *all* unsupported archs (which are a lot and adds maintenance burden, so please don't) or duplicate the whitelist here to be explicit in sync with the limitation imposed by llvm12, therefore make it easy to maintain and technically correct

Reply:
Thank you, I think it is correct now.